js/preload/preloadMain.js

```diff
@@ -23,6 +23,9 @@ const { TitleBar } = require('../windowsTitlebar');
 const titleBar = new TitleBar();
 const { buildNumber } = require('../../package.json');
 const SnackBar = require('../snackBar').SnackBar;
+const fs = remote.require('fs');
```

Comment:
This is a security concern. We'll be exposing entire `fs` to preload right? Isn't there an alternative using `IPC`?

Reply:
Currently, I don't think there are any solutions as IPC don't support buffers